packages/apostrophe/test/relationships.js

```diff
@@ -1,5 +1,6 @@
 const t = require('../test-lib/test.js');
 const assert = require('assert');
+const { findRelationships } = require('../modules/@apostrophecms/piece-type/ui/apos/lib/postprocessRelationships.js');
```

Comment:
I suppose the modern nodejs supports requiring ESM. I also hope we don't support officially old node versions without that support or the tests will fail.